docs/book/orders/orders.rst

```diff
@@ -195,8 +195,8 @@ Remember to process the order using order processor and then flush the order man
 
     $order->addPayment($payment);
 
-Shipping a Shipment with a state machine transition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Completing a payment with state machine transition
```

Comment:
Why `Shipping a Shipment` has two letters capitalized and `Completing a payment` has only one?